src/pages/Landing/Landing.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react'
+import { Box, Button, Container, Typography } from '@mui/material'
+import { Link as RouterLink } from 'react-router-dom'
+
+import { Contact } from './Contact'
+
+
+type ArticleType = {
+	title: string,
+	content: string,
+};
+
+const articles: ArticleType[] = [
+	{
+		title: 'Lorem Ipsum',
+		content:
+"Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s",
+	},
+	{
+		title: 'Lorem Ipsum',
+		content:
+"Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s",
+	},
+	{
+		title: 'Lorem Ipsum',
+		content:
+"Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s",
+	},
+]
+
+const Article: React.FC<ArticleType> = ({ title, content }) => (
+	<Box
+		component="section"
+		sx={{
+			width: 'calc(33.33% - 20px)', // 1/3 width minus padding and margin
+			backgroundColor: '#1C140F',
+			color: '#fff',
+			borderRadius: '15px',
+			padding: '50px',
+			margin: '10px',
+		}}
+	>
+		<Typography component="h2" mb='10px' sx={{fontSize: '25px', marginBottom: '40px'}}>{title}</Typography>
+		<Typography component="p"  sx={{fontSize: '15px', lineHeight: '1.8'}}>{content}</Typography>
+	</Box>
+)
+
+const ButtonStyle1 = {
+	width: 'calc(45% - 20px)', padding: '10px', margin: '10px', borderRadius: '25px', height: '50px', backgroundImage: 'linear-gradient(135deg, #FA2705, #FB9E3C)'
+}
+
+const ButtonStyle2 = {
+	width: 'calc(45% - 20px)', padding: '10px', margin: '10px', borderRadius: '25px', height: '50px', backgroundImage: 'linear-gradient(135deg, #7F23F5, #2824F5)',
+}
+
+
+export const Landing: React.FC = () => {
+	return (
+		<>
+			<div style={{ backgroundColor: '#39289F', padding: '5vw'}} id='landing-main'>
```

Comment:
with lots of styles here, it's  better to be create a styled component for this div

Reply:
done